README.md

```diff
@@ -58,234 +29,151 @@ For more information on the importance of a professional README for your reposit
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Fullstack Personal Portfolio] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- [optional] kindly consider removing this as it was given as guideline removing it will make your readme look professional